Benchmark/System/Process.hs

```diff
@@ -16,11 +18,21 @@ import System.Process (proc, createProcess, waitForProcess, callCommand)
 import qualified Streamly.Prelude as S
 import qualified Streamly.System.Process as Proc
 import qualified Streamly.Data.Fold as FL
+import qualified Streamly.FileSystem.Handle as FH
 
 -- Internal imports
 import qualified Streamly.Internal.FileSystem.Handle
     as FH (toBytes, toChunks, putBytes, putChunks)
 
+-- XXX replace with streamly versions once they are fixed
+{-# INLINE rights #-}
+rights :: (S.IsStream t, Monad m, Functor (t m)) => t m (Either a b) -> t m b
+rights = fmap (fromRight undefined) . S.filter isRight
+
+{-# INLINE lefts #-}
+lefts :: (S.IsStream t, Monad m, Functor (t m)) => t m (Either a b) -> t m a
+lefts = fmap (fromLeft undefined) . S.filter isLeft
```

Comment:
What's wrong with them? I've used streamly versions in tests.

Reply:
There is no INLINE pragma, hurting performance.